api/v1alpha1/osbuildenvconfig_types.go

```diff
@@ -25,11 +26,126 @@ import (
 
 // OSBuildEnvConfigSpec defines the desired state of OSBuildEnvConfig
 type OSBuildEnvConfigSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	// Composer contains all the required configuration values for the Composer service
```

Comment:
Do you think we need to limit the version of the osbuild-composer?

Reply:
Not sure what you mean? The image tag and the OSBuild-Worker and OSBuild versions? If yes, then I don't think we want the user to control that at the CR level. It will create a support matrix from hell

Reply:
Lets talk about it in the sync meeting and open a relevant ticket if needed

Reply:
Resolving this conversation. If we decide to allow the user to set the version, it will be added in a separate PR